### PR TITLE
Adds weekly cron-entry to update AD computerObject

### LINKED ISSUE #201: [Enhancement] Ensure that AD ComputerObject Is Regularly Refreshed
Vendor documentation recommends that AD-joined systems be configured to regularly update their computerObject information in active directory. Need to add either a `systemd` unit file &ndash; configured to run at each system boot &ndash; or a regularly-executed `cron` job to handle the task.

Given that domain-joined systems are typically not rebooted as frequently as the desired refresh interval (less than 30 days), adding an `/etc/cron.{daily,weekly,monthly}`  script with contents similar to:

~~~bash
source /etc/os-release
/usr/sbin/adcli update \
  --os-name="${NAME}" \
  --os-version="${VERSION}" \
  --os-service-pack="${VERSION_ID}"
~~~

Would be the preferred method for ensuring adequately-frequent object-refreshing.

### PR DESCRIPTION
**Pull request by ferricoxide**
Closes #201

### CODE REVIEW

**Comment by ferricoxide** [comment]
@lorengordon is this project no longer associated with an @-mentionable group??

File: join-domain/elx/sssd/init.sls [hunk 1]
```diff
@@ -153,3 +153,22 @@ Ensure with-mkhomedir feature is enabled:
...
+  file.managed:
+    - name: /etc/cron.weekly/refreshComputerObject.sh
+    - contents: |-
+        #!/bin/bash
+
+        source /etc/os-release
+        /usr/sbin/adcli update \
+          --os-name="${NAME}" \
+          --os-version="${VERSION}" \
+          --os-service-pack="${VERSION_ID}"
```
Comment (lorengordon) [inline join-domain/elx/sssd/init.sls:166]: All of these attributes are gated by permissions in AD, and not every environment may allow the service account to set every value. [We already gate the `os-name` and `os-version`](https://github.com/plus3it/join-domain-formula/blob/d472f8a71ab5f663bc6de365d1d379f8bdcb3ea6/join-domain/elx/sssd/files/join.sh#L82-L90), making them optional. We do not yet set `os-service-pack` at all. I think this command needs to be built up to account for it being optional to set any one of these values...

**Comment by lorengordon** [comment]
> @lorengordon is this project no longer associated with an @-mentionable group??

All salt-formulas are mapped to @plus3it/salt-formulas ...

File: join-domain/elx/sssd/init.sls (hunk 1 above)
Comment (ferricoxide) [reply join-domain/elx/sssd/init.sls:166]: This is a lifecycle, task, not a provisioning-time task: no service account is used; the computerObject, itself, is used to update the computerObject's _own_ AD attributes. I don't know that a computerObject _can_ be configured to disallow updating its own attributes (without breaking things).

Comment (lorengordon) [reply join-domain/elx/sssd/init.sls:166]: I suppose worst case, it's a cron that's failing, not the join action...

**Review by lorengordon — APPROVED** [review]

**Pull request merged**